collection-roller/src/main/scala/io/phdata/pulse/collectionroller/CollectionRollerMain.scala

```diff
@@ -79,6 +81,28 @@ object CollectionRollerMain extends LazyLogging {
     val (solr, solrService, collectionRoller) = createServices(parsedArgs)
     try {
       collectionRoller.collectionList().foreach(println)
+
+    } finally {
+      solrService.close()
+      solr.shutdown()
+    }
+    logger.info("Ending Collection Roller List App")
+  }
+
+  private def listApplicationsVerbose(parsedArgs: CollectionRollerCliArgsParser): Unit = {
+    logger.info("Starting Collection Roller List App")
+
+    val (solr, solrService, collectionRoller) = createServices(parsedArgs)
+    try {
+
+      println("Applications:")
+      collectionRoller.collectionList().foreach(println)
+      println("Aliases/Collections:")
+
+      for ((k, v) <- collection.mutable.LinkedHashMap(
```

Comment:
I don't think creating the mutable hash map is necessary here, try calling `toMap` on the sorted sequence instead